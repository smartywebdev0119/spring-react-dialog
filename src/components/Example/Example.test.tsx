import { Example } from './Example'
import { render } from '@testing-library/react'

describe('Test', () => {
  test('Snapshot', () => {
    const { container } = render(<Example />)

    expect(container).toMatchSnapshot()
  })
})
