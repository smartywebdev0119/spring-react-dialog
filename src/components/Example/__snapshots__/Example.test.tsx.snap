// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Snapshot 1`] = `
<div>
  <div>
    Example component
  </div>
</div>
`;
